content/blog/getting-started-to-generate-dscc-api-client-libraries-for-python-using-openapi-generator.md

```diff
@@ -0,0 +1,217 @@
+
+1. Provide an authorized token which is generated from the HPE GreenLake as mentioned in the [blog](https://developer.hpe.com/blog/oauth2-for-hpe-greenlake-data-services-cloud-console/)
+2. Install the Python dependencies using the following command.
+
+```shell
+~$ pip install requirements.txt 
+```
+
+Below is the sample of the code to get the information about the associated user's RBAC association. This code will provide list of the capabilities (port.read, volume.create) of that related user who provided the authorization can exercise. To execute this code, please substitute the YOUR_BEARER_TOKEN with the access token generated in your example.
+
+```python
+import time
+import openapi_client
+from openapi_client.api import authz_api
+from openapi_client.model.error_response import ErrorResponse
+from openapi_client.model.access_controls_response import AccessControlsResponse
+from pprint import pprint
+# Defining the host is optional and defaults to https://eu1.data.cloud.hpe.com
+# See configuration.py for a list of all supported configuration parameters.
+configuration = openapi_client.Configuration(
+    host = "https://eu1.data.cloud.hpe.com"
+)
+# The client must configure the authentication and authorization parameters
+# in accordance with the API server security policy.
+# Examples for each auth method are provided below, use the example that
+# satisfies your auth use case.
+
+# Configure Bearer authorization (JWT): JWTAuth
+configuration = openapi_client.Configuration(
+    access_token = 'YOUR_BEARER_TOKEN'
+)
+
+# Enter a context with an instance of the API client
+with openapi_client.ApiClient(configuration) as api_client:
+    # Create an instance of the API class
+    api_instance = authz_api.AuthzApi(api_client)
+    permission = ["volume.create","port.read","audit.read"] # [str] | List of permissions, each of which, has the form \"resource type.permission\" (ex. volume.read,volume.write). The word \"ANY\" can be used as a wild card for the resource type (ex. ANY.read). Omitting the permission parameters is equivalent to asking for all user permissions. (optional)
+
+    # example passing only required values which don't have defaults set
+    # and optional values
+    try:
+        # Get User Accessible Resources
+        api_response = api_instance.get_access_controls(permission=permission)
+        pprint(api_response)
+    except openapi_client.ApiException as e:
+        print("Exception when calling AuthzApi->get_access_controls: %s\n" % e)
+```
+
+The output from the execution of the above code is shown below:
+
+```Shell
+$ python .\GetAudits.py
+{'items': ['port.read', 'volume.create']}
+```
+
+Using these client generator tools, client libraries for DSCC can be generated faster and can be pushed to a GitHub repository and be made ready to use. One good advantage is that, automation of this process can be done by using CI/CD pipelines which require no manual intervention in updating any latest released APIs. 
```

Comment:
Take out the word "good" in the second sentence. Take out the comma after "that". Add "the" prior to "automation". Change "which" to "that". Add in "of the" prior to "latest".
So it reads:
"One advantage is that the automation of this process can be done by using CI/CD pipelines that require no manual intervention in updating any of the latest released APIs."